Dockerfile

```diff
@@ -3,12 +3,12 @@ FROM golang:1.17.6 as builder
 # Clone repositories
 WORKDIR /repos
 RUN git clone https://github.com/ElrondNetwork/rosetta-docker-scripts.git --branch=v0.2.1 --depth=1
-RUN git clone https://github.com/ElrondNetwork/elrond-config-devnet --branch=D1.3.44.0-rosetta3 --depth=1
-RUN git clone https://github.com/ElrondNetwork/elrond-config-mainnet --branch=v1.3.44.0-rosetta2 --depth=1
+RUN git clone https://github.com/ElrondNetwork/elrond-config-devnet --branch=D1.3.48.0-hf-fix --depth=1
```

Comment:
Still, the `v1.3.48.0` will be used. 

Mainnet vs. devnet binaries will be separated in https://github.com/ElrondNetwork/rosetta-docker/pull/16.